src/Wrapper.ts

```diff
@@ -51,11 +51,11 @@ const propTypes = {
 export interface WrapperProps {
   innerRef?: (ref: any) => void;
   name: string;
-  required: RequiredValidation;
-  validationError: any;
-  validationErrors: any;
-  validations: Validations | string;
-  value: any;
+  required?: RequiredValidation;
```

Comment:
Do you a have a reason to keep `RequiredValidation = string | true | Validations`?

Having "true" instead of "boolean" is blocking me from using booleans because "false" is not assignable to "true".
Did you had a reason behind this? Maybe changing it to boolean might be better. Unless I'm doing something wrong.

Reply:
No, that is a great point.